hazelcast/src/main/java/com/hazelcast/client/console/HazelcastCommandLine.java

```diff
@@ -189,9 +189,10 @@ public void submit(
             printf("Will restore the job from the snapshot with name '%s'", snapshotName);
         }
 
+        // calledByMember is false
```

Comment:
not necessary, especially that IntelliJ already is adding description of field

Reply:
Clarified the comment explaining why were passing false value